[2025-08-05T19:23:45.612+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_medallon.ejecutar_merge manual__2025-08-05T19:17:56.307282+00:00 [queued]>
[2025-08-05T19:23:45.644+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_medallon.ejecutar_merge manual__2025-08-05T19:17:56.307282+00:00 [queued]>
[2025-08-05T19:23:45.645+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-08-05T19:23:45.716+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): ejecutar_merge> on 2025-08-05 19:17:56.307282+00:00
[2025-08-05T19:23:45.720+0000] {standard_task_runner.py:60} INFO - Started process 247 to run task
[2025-08-05T19:23:45.724+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_medallon', 'ejecutar_merge', 'manual__2025-08-05T19:17:56.307282+00:00', '--job-id', '260', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_medallon_dag.py', '--cfg-path', '/tmp/tmpyc0su07g']
[2025-08-05T19:23:45.726+0000] {standard_task_runner.py:88} INFO - Job 260: Subtask ejecutar_merge
[2025-08-05T19:23:45.815+0000] {task_command.py:423} INFO - Running <TaskInstance: pipeline_medallon.ejecutar_merge manual__2025-08-05T19:17:56.307282+00:00 [running]> on host 70eaa744a05c
[2025-08-05T19:23:45.991+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mateo.biasotto' AIRFLOW_CTX_DAG_ID='pipeline_medallon' AIRFLOW_CTX_TASK_ID='ejecutar_merge' AIRFLOW_CTX_EXECUTION_DATE='2025-08-05T19:17:56.307282+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-05T19:17:56.307282+00:00'
[2025-08-05T19:23:45.992+0000] {sql.py:276} INFO - Executing: CALL merge_bulk_to_final();
[2025-08-05T19:23:46.007+0000] {base.py:83} INFO - Using connection ID 'postgres_medallon' for task execution.
[2025-08-05T19:23:46.097+0000] {base.py:83} INFO - Using connection ID 'postgres_medallon' for task execution.
[2025-08-05T19:23:46.137+0000] {sql.py:450} INFO - Running statement: CALL merge_bulk_to_final();, parameters: None
[2025-08-05T19:23:46.749+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=pipeline_medallon, task_id=ejecutar_merge, execution_date=20250805T191756, start_date=20250805T192345, end_date=20250805T192346
[2025-08-05T19:23:46.820+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-05T19:23:46.846+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
